spread.yaml

```diff
@@ -25,11 +25,11 @@ environment:
     ARCH/rpi: armhf
     RELEASE: focal
     RELEASE/ubuntu: focal
-    RELEASE/ubuntu_devel: groovy
-    RELEASE/ubuntu_bionic: bionic
+    RELEASE/ubuntu_devel: hirsute
+    RELEASE/ubuntu_groovy: groovy
     RELEASE/debian_sid: sid
     MIR_PPA: false
-    MIR_PPA/ubuntu_bionic: true
+    MIR_PPA/ubuntu_focal: true
```

Comment:
This should actually go away altogether, along with the adding of the PPA in `spread/…`.

Note `/ubuntu_focal` here wouldn't match anything.

Reply:
Done